src/vs/workbench/contrib/files/browser/fileCommands.ts

```diff
@@ -566,7 +566,12 @@ CommandsRegistry.registerCommand({
 
 CommandsRegistry.registerCommand({
 	id: REMOVE_ROOT_FOLDER_COMMAND_ID,
-	handler: (accessor, resource: URI | object) => {
+	handler: async (accessor, resource: URI | object) => {
+		// MEMBRANE: We override this functionality in vscode-extension
+		const commandService = accessor.get(ICommandService);
+		const name = 'path' in resource ? resource.path.slice(1) : undefined;
+		commandService.executeCommand('membrane.deleteProgram', { name });
```

Comment:
This requires membrane-io/mnode#163. It seems like the easiest option we have is to simply bypass vscode's handling and just do what we'd do in our extension.